ts_src/psbt.ts

```diff
@@ -0,0 +1,1160 @@
+import { Psbt as PsbtBase } from 'bip174';
+import * as varuint from 'bip174/src/lib/converter/varint';
+import {
+  NonWitnessUtxo,
+  PartialSig,
+  PsbtInput,
+  TransactionInput,
+  TransactionOutput,
+} from 'bip174/src/lib/interfaces';
+import { checkForInput } from 'bip174/src/lib/utils';
+import { toOutputScript } from './address';
+import { reverseBuffer } from './bufferutils';
+import { hash160 } from './crypto';
+import {
+  fromPublicKey as ecPairFromPublicKey,
+  Signer,
+  SignerAsync,
+} from './ecpair';
+import { bitcoin as btcNetwork, Network } from './networks';
+import * as payments from './payments';
+import * as bscript from './script';
+import { Output, Transaction } from './transaction';
+
+const DEFAULT_OPTS: PsbtOpts = {
+  network: btcNetwork,
+  maximumFeeRate: 5000, // satoshi per byte
+};
+
+export class Psbt extends PsbtBase {
```

Comment:
I think `Psbt` does not have to inherit `PsbtBase`.
Having a `PsbtBase` instance as a property and calling methods of it is enough to manage PSBT format, isn't it?
That also complies with the [Composition over inheritance](https://en.wikipedia.org/wiki/Composition_over_inheritance) principle.

Reply:
I'll try it out and see.

Reply:
I tried it out... it's kind of long... :-/